site/utils/index.ts

```diff
@@ -7,3 +10,6 @@ export const buildTitle = (errors: ErrorInfo[], title: string): string => {
 
     return title;
 };
+
+export const getCsrfToken = (ctx: DocumentContext | NextPageContext | NextPageContext): string =>
```

Comment:
I think we can probably get rid of DocumentContext here

Reply:
done